concent_api/requirements.txt

```diff
@@ -1,5 +1,5 @@
 Django >= 1.11, < 1.12
 psycopg2
--e git://github.com/golemfactory/golem-messages.git@v1.1.0#egg=golem_messages
+-e git://github.com/Code-Poets/golem-messages.git@303ff545e9db7631b078ada2f8a9a4b336c39b30#egg=Golem_Messages
```

Comment:
It's preferable to use branch name in `requirements.txt`. Unless you really really need only this particular commit which, I think, is not the case here. `requirements.lock` will get a commit anyway.

Reply:
Now you have no version at all. You should add a branch name.

Reply:
FileTransferToken has been pulled to master, so I changed version to:
`-e git://github.com/golemfactory/golem-messages.git@master#egg=golem_messages`